library/std/src/sys/unix/thread_parking/netbsd.rs

```diff
@@ -0,0 +1,54 @@
+#![cfg(target_os = "netbsd")]
+
```

Comment:
Why is this cfg here? The `mod netbsd;` in mod.rs already has this cfg, right?

Reply:
Yeah, that's unnecessary. I don't know why I added that :shrug: